sdk/kusto/Azure.ResourceManager.Kusto/CHANGELOG.md

```diff
@@ -1,15 +1,17 @@
 # Release History
 
-## 1.2.0-beta.1 (Unreleased)
+## 1.2.0 (2023-01-03)
```

Comment:
Please give 2 weeks ahead at least to release a new package

Reply:
@ArthurMa1978 
2 weeks is (2023-01-17)? 
Can you please explain why the 2 weeks ahead is needed?
How soon can I get a dev feed nugget?

Reply:
updated to 2023-01-17